packages/react/src/ActionList/ActionList.module.css

```diff
@@ -213,6 +213,22 @@
       }
     }
 
+    &:where([data-last-selected]:not(:last-of-type)) {
+      padding-bottom: var(--base-size-4);
```

Comment:
Why do we reduce the padding for these items?

Reply:
I'm actually adding padding to separate from the divider line, this is only added to the last selected item for such purposes

Reply:
I still don't fully understand why it's 4px instead of 6px 🙈 

Maybe we you can screen-share and explain?